manifests/params.pp

```diff
@@ -53,6 +53,8 @@
   $gpgcheck    = true
   $version     = 'present'
 
+  $noderb_timeout = 10
```

Comment:
We should probably increase this default to avoid the Passenger startup issue you describe?